backend/app/views/spree/admin/images/_image_row.html.erb

```diff
@@ -7,9 +7,15 @@
   </td>
 
   <td>
-    <%= link_to image.attachment.url(:product) do %>
+    <span class="modal-image-trigger" data-toggle="modal" data-target="#modal-image-<%= image.id %>">
       <%= render 'spree/admin/shared/image', image: image, size: :mini %>
-    <% end %>
+    </span>
+    <%= render(
+      "spree/admin/shared/modal",
+      target: "modal-image-#{image.id}",
+      title: image.alt,
+      content: image_tag(image.attachment.url(:large))
```

Comment:
Just my personal taste, but wouldn’t it be nice to wrap the image in `div` that centers the image?